src/tools/scripts/migration-role-hub.ts

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env node
+/* eslint-disable no-await-in-loop */
+import execute from '../utils';
+import service from '../../helpers/services';
+
+const { v4: uuidv4 } = require('uuid');
+
+// ts-node src/tools/scripts/migration-role-hub.ts
+
+execute(__filename, async ({ app, logger, exit, Sentry }) => {
+  try {
+    const users = await app.service(service.users).Model.find({
+      roles: { $in: ['hub_coop'] },
+    });
+
+    logger.info(`Nombre d'utilisateurs à traiter: ${users?.length}`);
+
+    for (const user of users) {
+      if (user.passwordCreated === true) {
```

Comment:
Est-ce qu'on peut simplifier comme ça ?
```suggestion
      if (user.passwordCreated) {
```